pkg/bindings/bindings.go

```diff
@@ -11,8 +11,10 @@ package bindings
 var (
 	// PTrue is a convenience variable that can be used in bindings where
 	// a pointer to a bool (optional parameter) is required.
-	PTrue bool = true
+	pTrue = true
+	PTrue = &pTrue
 	// PFalse is a convenience variable that can be used in bindings where
```

Comment:
PFalse and PTrue in comments need to change to pFalse and pTrue

Reply:
oops, never mind, I should look at line two for each of these.

Reply:
@TomSweeneyRedHat The comments are more meaningful now. We don't want people using pTrue or pFalse.